packages/generators/add-generator.ts

```diff
@@ -98,17 +98,19 @@ export default class AddGenerator extends Generator {
 				webpackOptions: {}
 			}
 		};
-		const { registerPrompt } = this.env.adapter.promptModule;
-		registerPrompt("autocomplete", autoComplete);
+		// TODO adapter doesn't exists
+		// const { registerPrompt } = this.env.adapter.promptModule;
+		// registerPrompt("autocomplete", autoComplete);
```

Comment:
Can anyone explain me what is this? :)

Reply:
@dhruvdutt might know something

Reply:
Some context, the type of env in [DefinitelyTyped ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/yeoman-generator/index.d.ts#L89) is:

```
    env: {
        error(...e: Error[]): void
    };
```

Reply:
The adapter is the part that actually handler the user interaction ([doc](https://yeoman.io/authoring/integrating-yeoman.html)).
`registerPrompt` is used to define a custom Inquirer prompt ([source](https://github.com/SBoudrias/Inquirer.js/blob/master/packages/inquirer/lib/inquirer.js#L46)).

I'm going to open a PR to include those in DefinitelyTyped, I just need to understand how to write tests :)